recruiter/models.py

```diff
@@ -35,3 +35,14 @@ class Recruiter(models.Model):
 
     def __str__(self):
         return f'{self.name}'
+
+    @classmethod
+    def is_recruiter(cls, auth_user_id):
+        """
+        For checking if a currently logged in user is a Recruiter
+        """
+        # Get the Recruiter instance associated with the given authentication user
+        associated_recruiter_instance = cls.objects.filter(user_id=auth_user_id)
+        if associated_recruiter_instance:  # If true, a matching Recruiter was found
+            return True
+        return False
```

Comment:
I think you can combine these 4 lines into 1 - `return cls.objects.filter(user_id=auth_user_id).exists()`

Reply:
Thanks! done.